electron/ui/src/tests/FlowsheetsListTable.test.js

```diff
@@ -0,0 +1,13 @@
+import { render, screen } from '@testing-library/react';
+import FlowsheetsListTable from "../components/FlowsheetsListTable/FlowsheetsListTable"
+import test_data from '../../../../backend/data/flowsheets/fake/data.json'
+import * as React from 'react'
+
+test('test flowsheets list table', () => {
```

Comment:
The METAB flowsheet get moved/renamed at some point. Not sure how to deal with that automatically